src/index.js

```diff
@@ -100,13 +104,14 @@ const VueCroppie = {
         },
         result(options, cb) {
           if(!options) options = {type: 'base64'}
-            this.croppie.result(options).then(output => {
+            return this.croppie.result(options).then(output => {
               if(!cb) {
                 this.$emit('result', output);
               } else {
                 cb(output);
               }
               this.refresh();
+              return output
```

Comment:
I think the `return output` here might be unnecessary because the if block above it is already returning either a callback or an event.

Reply:
Based on your point that we should return the promise. I think it'd be better like this? 

```
const promise = this.croppie.result(options);
if(!cb) {
  return promise;
} else {
  promise.then(output => cb(ouput));
}
```

But this will completely omit the `this.$emit('result', output);` which is I think more important than the cb. What do you think?

Reply:
I did it like that so we can keep backwards compatibility and still have promises. Pretty much you are free to choose between a callback and a promise resolution. If we dont return it, the next `then()` will have no value :)

Reply:
@dobromir-hristov It makes sense for backwards compatibility. :) Maybe we need to make this clear in the next versions. We need choose between the two. Thanks.

Reply:
Currently, in our company implementation, we are bypassing most of the helper methods from the wrapper and directly using the croppie instance, which was kind of stupid in my mind, but we do it because we chain allot of promises, and as the croppie library offers promises it was kind of a drawback. 
In my opinion supplying a callback is pretty much the same, just a tad bit messier, but we shouldnt just remove it as it will be a breaking change.

Reply:
@dobromir-hristov I agree. Maybe let's just "prettify" the code for the next version.

Reply:
@dobromir-hristov thank you for the contribution.